docs/dev/explain-packaging-data.md

```diff
@@ -50,7 +50,7 @@ In 2020, we made this field a taxonomized field. As a result, we now store the l
 
 #### Packaging information / recycling instructions text field (READ and WRITE)
 
-In 2020, we also added a language specific field ("packaging_text_[language code]" e.g. "packaging_text_en" for English) to store free text data about the packaging. It can contain the text of the recycling instructions printed on the packaging (e.g. "bottle to recycle, cap to discard"), or can be filled in by users (e.g. "1 PET plastic bottle to recycle, 1 plastic cap").
+In 2020, we also added a language specific field ("packaging*text*[language code]" e.g. "packaging_text_en" for English) to store free text data about the packaging. It can contain the text of the recycling instructions printed on the packaging (e.g. "bottle to recycle, cap to discard"), or can be filled in by users (e.g. "1 PET plastic bottle to recycle, 1 plastic cap").
```

Comment:
The underscore are part of the name of the variable, they should not be replaced with *, otherwise it removes the underscore and makes some text italic:

![image](https://user-images.githubusercontent.com/8158668/229448198-84815e15-a2a4-4857-995a-b65196a432cb.png)